src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -1,6 +1,7 @@
 /*
  * MCreator (https://mcreator.net/)
- * Copyright (C) 2020 Pylo and contributors
+ * Copyright (C) 2012-2020, Pylo

```

Comment:
Checking JS bridge, do we still use `getReadableListOfForWorkspace`, `getReadableListOf`?